mods/taskbar-autohide-better.wh.cpp

```diff
@@ -0,0 +1,105 @@
+// ==WindhawkMod==
+// @id              taskbar-autohide-better
+// @name            Better Taskbar Autohide
+// @description     Allow taskbar autohide when inactive window notified
+// @version         1.0
+// @author          Cirn09
+// @github          https://github.com/cirn09
+// @homepage        https://blog.cirn09.xyz/
+// @include         explorer.exe
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# Better Taskbar Autohide
+Allow taskbar autohide when inactive window notified
+![](https://p.sda1.dev/13/f403002dd34d5e45de384eb8eb626922/demo.gif)
```

Comment:
Only images from the following domains can be embedded in the readme: https://i.imgur.com/, https://raw.githubusercontent.com/. See [the wiki](https://github.com/ramensoftware/windhawk/wiki/Creating-a-new-mod#readme).

You can compile the mod and click on Preview Mod to make sure it looks correctly.